AirPlan/AirPlan/View/LoginView.swift

```diff
@@ -17,13 +17,15 @@ enum LoginStatus {
 }
 
 class LoginView: UIView, UIEssentials {
+    let loginStackView = UIStackView()
     let idTextField = UITextField()
     let passwordTextField = UITextField()
     let loginButton = UIButton()
     let signUpButton = UIButton()
     var signInGoogleButton = GIDSignInButton()
-    let testLabel = UILabel()
     let testLogOutButton = UIButton()
+    let logoImage = UIImageView()
+    let infoText = UILabel()
```

Comment:
```suggestion
    let infoLabel = UILabel()
```

suffix로 컴포넌트의 이름을 붙여주세요!